api/tests/unit/domain/usecases/create-tag_test.js

```diff
@@ -0,0 +1,21 @@
+const { expect, domainBuilder, sinon } = require('../../../test-helper');
+const Tag = require('../../../../lib/domain/models/Tag');
+const createTag = require('../../../../lib/domain/usecases/create-tag');
+
+describe('Unit | UseCase | create-tag', function () {
+  it('should allow to create an tag', async function () {
```

Comment:
suggestion `should allow to create a tag`

Reply:
done !